core/embed/rust/src/ui/model_tr/trace.rs

```diff
@@ -0,0 +1,19 @@
+use crate::trace::Tracer;
+
+use super::component::ButtonPos;
+
+use heapless::String;
+
+pub trait ButtonTrace {
```

Comment:
could we throw out this whole thing?

it's causing spammy code and polluting usages of `ButtonAction` (which are otherwise limited to the Flow component, it seems?) because in a lot of places you implement `get_btn_action -> String` like "if some condition then `ButtonAction::NextPage.string()` (where you could just as easily say `"NextPage"` instead)

istm that "report btn actions" is only relevant for Flow components, because in the others it's either (a) obvious, or (b) misleading (there are no ButtonActions involved but they get placed in the trace?), and/or (c) made redundant by reporting contents of the ButtonLayout

Reply:
I do not think we can easily get rid of this. The biggest usage is in `input_methods`, where it reports what is the current choice - "what happens when I press middle button". This information is not part of `ButtonLayout`, which just focuses on button visuals.

Changed significantly in c41ae886c8e5dfa6154a276e77490378686ab9a1

Reply:
Removed completely